ios/ReactViewPagerManager.m

```diff
@@ -66,6 +66,18 @@ - (void) changeScrollEnabled
     [self changeScrollEnabled:reactTag enabled:isEnabled];
 }
 
```

Comment:
Should `RCT_EXPORT_METHOD(setPage` and `RCT_EXPORT_METHOD(setPageWithoutAnimation` be deleted?

Reply:
If we gonna keep it for backward compatibility need to reflect it in Readme